assets/admin/assets/js/chart.morris.js

```diff
@@ -0,0 +1,55 @@
+$(function(){
+	
+	/* Morris Area Chart */
+	
+	window.mA = Morris.Area({
+	    element: 'morrisArea',
+	    data: [
+	        { y: '2013', a: 60},
+	        { y: '2014', a: 100},
+	        { y: '2015', a: 240},
+	        { y: '2016', a: 120},
+	        { y: '2017', a: 80},
+	        { y: '2018', a: 100},
+	        { y: '2019', a: 300},
+	    ],
+	    xkey: 'y',
+	    ykeys: ['a'],
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/BuildForSDGCohort2/Team-317-Doctor-Appointment/pullRequest?prid=6207082)